nixos/modules/services/web-apps/dolibarr/preinstalled-db.sql

```diff
@@ -0,0 +1,12228 @@
+-- MariaDB dump 10.19  Distrib 10.6.8-MariaDB, for Linux (x86_64)
```

Comment:
This is a quite huge file. I do not think it would be a good idea to add this to already bloated nixpkgs. Can this instead be fetched from an external location using a fetcher derivation?

Reply:
Or alternatively generated?

Reply:
Yes, we must do something about this file.